main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
```

Comment:
But even with using `_.template()`, we still may need concatenation because the string is so long and stuffing it on a single line is going to be a real pain and not very editable. I would honestly rather use concatenation everywhere rather than just half and half.